k8s/main.go

```diff
@@ -202,8 +229,8 @@ func main() {
 	var schedulerConfigFile string
 	var enableLeaderElection bool
 
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
-	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8089", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8090", "The address the probe endpoint binds to.")
 	flag.StringVar(&schedulerConfigFile, "scheduler-config-file", "/etc/kubernetes/scheduler.yaml", "The location of scheduler plugin's configuration file.")
```

Comment:
Why this change? They should be the default generation.

Reply:
Oh, it's my fault, Thanks for the catch, I use it for testing locally and forget to change it back.